style.css

```diff
@@ -558,3 +473,346 @@ input[type="submit"]:disabled {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .work1-list li {
+    padding: 10px 12px;
+    font-size: 15px;
+  }
+
+  .work-menu {
+    margin-bottom: 30px;
+  }
+
+  .p {
+    margin-left: -40px;
+  }
+
+  .btn {
+    padding: 12px;
+    font-size: 17px;
+    line-height: 24px;
+    margin-top: 60px;
+  }
+
+  .gridy {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    grid-template-rows: auto auto auto;
+    gap: 15px;
+  }
+
+  .work-title2a {
+    background-image: linear-gradient(180.45deg, rgb(38 38 38 / 0%) 0.75%, rgb(38 38 38 / 90%) 84.18%), url("img/Img\ \(1\).png");
+  }
+
+  .go {
+    visibility: hidden;
+  }
+
+  .be {
+    background-image: linear-gradient(179.35deg, rgb(38 38 38 / 0%) 0.85%, rgb(38 38 38 / 90%) 84%), url("img/Img\ \(2\).png");
+  }
+
+  .bee {
+    background-image: linear-gradient(179.4deg, rgb(38 38 38 / 0%) 0.81%, rgb(38 38 38 / 90%) 84.03%), url("img/Img\ \(3\).png");
+  }
+
+  .be a,
+  .bee a {
+    display: none;
+  }
+
+  .bee h1 {
+    margin-top: 20px;
+  }
+
+  .work-he {
+    color: white;
+    line-height: 44px;
+    font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial, sans-serif;
+    font-weight: 400;
+    font-size: 32px;
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+
+  .work-head22 {
+    visibility: hidden;
+  }
+
+  .work2-para {
+    font-size: 17px;
+    line-height: 24px;
+  }
+
+  .bet li {
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .about {
+    padding: 160px;
+    padding-right: 0;
+  }
+
+  .about-title {
+    font-size: 72px;
+    line-height: 82px;
+  }
+
+  .about-para {
+    width: 566px;
+    height: 148px;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .btn2 {
+    padding: 12px;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+  }
+
+  .block1 {
+    font-family: Arial, Helvetica, sans-serif;
+    display: none;
+  }
+
+  .block2 {
+    position: absolute;
+    right: 0;
+    top: 1.79%;
+    width: 40%;
+  }
+
+  .lang {
+    flex-direction: row;
+    gap: 15%;
+    align-items: initial;
+    padding: 5% 0;
+  }
+
+  .big,
+  .small {
+    text-align: left;
+  }
+
+  .contact {
+    padding: 60px 160px;
+  }
+
+  .cont {
+    display: flex;
+    flex-direction: row;
+    gap: 4%;
+  }
+
+  .contact-text {
+    width: 450px;
+    height: 18px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: left;
+  }
+
+  .input {
+    width: 700px;
+    height: 18px;
+    left: 16px;
+    right: 16px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+  }
+
+  .submit {
+    margin: 15px 0;
+  }
+
+  .last {
+    padding: 20px 45%;
+  }
+}
+
+@media screen and (max-width: 700px) {
+  body {
+    overflow-x: hidden;
```

Comment:
[OPTIONAL] Nice job adding media queries to your project :smiley:! Although, please take into account that, currently, you have one query with `max-width: 700px` and another one with `min-width: 768px`. This means that, between these values, your project doesn’t contain any of these stylings: 

![image](https://user-images.githubusercontent.com/42799579/206631661-fcd90a11-f034-4421-8ba8-07dbab46aaeb.png)

Please make sure to update your queries to wrap all the possible widths :wink:.